test/mock/config/manager.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import "github.com/NordSecurity/nordvpn-linux/config"
+
+type ConfigManagerMock struct {
```

Comment:
There is already a mock for ConfigManager, doesn't that work? `mock.ConfigManager` in `test/mock/config_manager.go`